### PR TITLE
Adds documentation on PKCS12 support for csi-driver

### PR DESCRIPTION
**Pull request by JoshVanL**
Code PR found [here](https://github.com/cert-manager/csi-driver/pull/103)

Also adds SERVICE_ACCOUNT unix variable support to next-docs.

/hold

Until code PR is merged and the feature is part of a release.

### CODE REVIEW

File: content/docs/projects/csi-driver.md [hunk 1]
```diff
@@ -135,25 +135,28 @@ The csi-driver driver aims to have complete feature parity with all possible
...
+| `csi.cert-manager.io/renew-before`             | The time to renew the certificate before expiry. Defaults to a third of the requested duration.                            | `$CERT_DURATION/3`                   | `72h`                            |
+| `csi.cert-manager.io/reuse-private-key`        | Re-use the same private when when renewing certificates.                                                                   | `false`                              | `true`                           |
+| `csi.cert-manager.io/keystore-pkcs12-enable`   | Enable writing the signed certificate chain and private key as a PKCS12 file.                                              |                                    | `true`                           |
+| `csi.cert-manager.io/keystore-pkcs12-file`     | File location to write the PKCS12 file. Requires `csi.cert-manager.io/keystore-pkcs12-file` be set to `true`.                  | `keystore.p12`                       | `tls.p12`                        |
```
Comment (jahrlin) [inline content/docs/projects/csi-driver.md:158]: This looks incorrect

```suggestion
| `csi.cert-manager.io/keystore-pkcs12-file`     | File location to write the PKCS12 file. Requires `csi.cert-manager.io/keystore-pkcs12-enable` be set to `true`.                  | `keystore.p12`                       | `tls.p12`                        |
```

File: content/next-docs/projects/csi-driver.md [hunk 2]
```diff
@@ -135,25 +135,28 @@ The csi-driver driver aims to have complete feature parity with all possible
...
+| `csi.cert-manager.io/renew-before`             | The time to renew the certificate before expiry. Defaults to a third of the requested duration.                            | `$CERT_DURATION/3`                   | `72h`                            |
+| `csi.cert-manager.io/reuse-private-key`        | Re-use the same private when when renewing certificates.                                                                   | `false`                              | `true`                           |
+| `csi.cert-manager.io/keystore-pkcs12-enable`   | Enable writing the signed certificate chain and private key as a PKCS12 file.                                              |                                    | `true`                           |
+| `csi.cert-manager.io/keystore-pkcs12-file`     | File location to write the PKCS12 file. Requires `csi.cert-manager.io/keystore-pkcs12-file` be set to `true`.                  | `keystore.p12`                       | `tls.p12`                        |
```
Comment (jahrlin) [inline content/next-docs/projects/csi-driver.md:158]:
```suggestion
| `csi.cert-manager.io/keystore-pkcs12-file`     | File location to write the PKCS12 file. Requires `csi.cert-manager.io/keystore-pkcs12-enable` be set to `true`.                  | `keystore.p12`                       | `tls.p12`                        |
```

**Review by jahrlin — APPROVED** [review]
Thanks @JoshVanL !

/lgtm
/approve

**Comment by JoshVanL** [comment]
Thanks for the catch @jahrlin, PTAL

**Comment by jahrlin** [comment]
Thanks for the fix

/lgtm

**Pull request merged**